.changelog/unreleased/features/3622-kvstore-mempool-lanes.md

```diff
@@ -1,2 +1,2 @@
 - `[kvstore]` Extended `CheckTx` in kvstoreApp to support mempool lanes. 
-  ([#3622](https://github.com/tendermint/tendermint/pull/3622))
+  ([#3622](https://github.com/cometbft/cometbft/pull/3622))
```

Comment:
```suggestion
  ([#3503](https://github.com/cometbft/cometbft/issues/3503))
```

Reply:
According to our contributing guidelines, PRs are preferred.